src/Zicht/Bundle/UrlBundle/Admin/StaticReferenceTranslationAdmin.php

```diff
@@ -34,7 +35,7 @@ protected function configureFormFields(FormMapper $form)
     {
         $form
             ->with('General')
-            ->add('locale', null, ['required' => true])
+            ->add('locale', LanguageType::class, ['required' => true])
```

Comment:
Dit gaat dus helemaal mis in projecten waar geen TranslationsBundle in zit... niet handig dit

Reply:
Volgens mij wordt de TranslationsBundle wel afgedwongen als requirement.  Wel goed punt datie dan nog geconfigureerd moet worden (al hebben we dat probleem niet meer met Symfony 4 scripts).

Ik ben wel een fan om dit zo te laten, want dit zorgt wel voor consistentie in het CMS

Reply:
> al hebben we dat probleem niet meer met Symfony 4 scripts

Die hebben we dus niet. Dus projecten gaan "stuk" hierdoor.

Ik heb deze aanpassing nu doorgevoerd in v4 en v5: https://github.com/zicht/url-bundle/commit/549122ccc164278df5a797e260201c74c41f7176